.github/workflows/linting.yml

```diff
@@ -18,3 +18,13 @@ jobs:
   call-workflow-passing-data:
     name: Documentation
     uses: RadeonOpenCompute/rocm-docs-core/.github/workflows/linting.yml@develop
+
```

Comment:
this section should move to rocm-docs-core after the test period is complete.

Reply:
https://github.com/RadeonOpenCompute/rocm-docs-core/pull/395